frontend/src/components/ArtistHeader.js

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components/macro'
+import {Link} from "react-router-dom";
+
+export default function ArtistHeader({artist}) {
+
+    return (
+        <Link to={`/artist/${artist.artistName}`} style={{textDecoration: 'none'}}>
+            <Header>
+                <img src={artist.artistImageUrl} alt={'Could not be loaded'}/>
```

Comment:
for accessibility purposes I would include an image description (e.g. artist name) in the `alt` tag, because it should include information for e.g. blind people who are not able to see your image, so it shouldn't be misused for error messages when an image could not be loaded properly

Reply:
thx. changed.